io.openrct2.OpenRCT2.yaml

```
@@ -22,6 +22,9 @@ finish-args:
   - --device=dri
   # Discord RPC
   - --filesystem=xdg-run/app/com.discordapp.Discord:create
+  # Access to host Steam-installed RCT and RCT2 for auto-detection in OpenRCT2
+  - --filesystem=~/.local/share/Steam/steamapps/common/Rollercoaster Tycoon Deluxe:ro
+  - --filesystem=~/.local/share/Steam/steamapps/common/Rollercoaster Tycoon 2:ro
```

The paths should probably use `xdg-data` instead of the hardcoded `~/.local/share` (from the [permissions docs](https://docs.flatpak.org/en/latest/sandbox-permissions.html#filesystem-access)).

```suggestion
  - --filesystem=xdg-data/Steam/steamapps/common/Rollercoaster Tycoon Deluxe:ro
  - --filesystem=xdg-data/Steam/steamapps/common/Rollercoaster Tycoon 2:ro
```

I'm assuming that Steam respects the `XDG_DATA_HOME` environment variable; if it ignores that and always uses `~/.local/share` then hardcoding it here as well is correct.

---

This is related to the OpenRCT2 code that tries to autodetect assets from the host Steam, and it doesn't use `$XDG_DATA_HOME`: https://github.com/OpenRCT2/OpenRCT2/blob/v0.4.11/src/openrct2/platform/Platform.Linux.cpp#L303

So `xdg-data` is indeed not needed.

---

OK, fair enough. :slightly_smiling_face: